texttospeech/snippets/audio_profile_test.go

```diff
@@ -0,0 +1,57 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package snippets
+
+import (
+	"bytes"
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/GoogleCloudPlatform/golang-samples/internal/testutil"
+)
+
+const (
+	text       = "hello"
+	outputFile = "output.mp3"
+)
+
+func TestAudioProfile(t *testing.T) {
+	testutil.SystemTest(t)
+
+	os.Remove(outputFile)
+
+	var buf bytes.Buffer
+	err := audioProfile(&buf, text, outputFile)
+	if err != nil {
+		t.Error(err)
```

Comment:
Fatal? Probably doesn't make sense to continue if this failed.

Reply:
Makes sense, changed